src/dmd/mars.d

```diff
@@ -486,6 +486,11 @@ private int tryMain(size_t argc, const(char)** argv)
         message("binary    %s", global.params.argv0);
         message("version   %s", global._version);
         message("config    %s", global.inifilename ? global.inifilename : "(none)");
+        OutBuffer buf;
+        foreach (flag; dflags.asDArray)
+            buf.printf("%s ", flag);
+        auto res = buf.peekSlice() ? buf.peekSlice()[0 .. $ - 1] : "(none)";
+        message("dflags    %.*s", res.length, res.ptr);
```

Comment:
Please use "DFLAGS" in all caps.

Reply:
I used the existing convention of lower-casing everything - the entire output of `-v` starts with lower-cases tags.

Reply:
Environment variables are case-sensitive (at least on Linux).  Therefore, I think the case is important here.

Reply:
What do other people think? (I don't mind changing it, but for now I focus on getting it to pass on auto-tester, so I won't change it immediately)

Reply:
doesn't matter to me, both have small pros/cons.  Lower case is consistent with all other verbose output, CAPITAL make it consistent with the actual variable name.